docs/appliance-config.md

```diff
@@ -0,0 +1,19 @@
+# Appliance Config
+* To generate a configuration `yaml`, please refer to [Generate a template of the appliance config](./user-guide.md#Generate-a-template-of-the-appliance-config)
+
+
+ Name                       | Default Value                  | Optional | Type    | Description
+----------------------------|--------------------------------|----------|---------|---------------------------------------------------
+ apiVersion                 | `v1beta1`                      | No       | string  | 
```

Comment:
As it's mandatory, we can keep the 'Default Value' empty, and explain in 'Description' to use 'v1beta1'. I.e. we don't set a default value for it in code.

Reply:
won't it fail marshaling without it?

Reply:
> won't it fail marshaling without it?

Yes, that's why 'default' is misleading I think. The value must be specified by user, but default implies to a fallback done automatically by code (e.g. as 'stable' for channel).

Reply:
So how about I'll leave the default value as 'blank', and in `Description` I'll just write: 
The configuration version that is currently supported by the appliance. options: `v1beta1`.

I'll also change the type to enum

Reply:
sounds good.

Reply:
done